Can't install with 64-bit Python3 on AIX 7.1

With Python3 from IBM's AIX Toolbox site, I can't install psutil. It fails to build the C code:
```
psutil/arch/aix/net_connections.c:63:15: error: 'struct mbuf' has no member named 'm_len'
         if (mb.m_len > sizeof(struct sockaddr_un))
               ^
psutil/arch/aix/net_connections.c:64:15: error: 'struct mbuf' has no member named 'm_len'
             mb.m_len = sizeof(struct sockaddr_un);
               ^
psutil/arch/aix/net_connections.c:65:26: error: 'struct mbuf' has no member named 'm_len'
         *((char *)ua + mb.m_len - 1) = '\0';
                          ^
```

This build of Python is 64-bit:
```
$ python3 -c 'import struct; print(struct.calcsize("P"))'
8
```

I can get it to build by editing `psutil/arch/aix/net_kernel_structs.h` to move the last `#define` outside of the conditional.
```
$ git diff | cat
diff --git a/psutil/arch/aix/net_kernel_structs.h b/psutil/arch/aix/net_kernel_structs.h
index 09f320ff..11e343d8 100644
--- a/psutil/arch/aix/net_kernel_structs.h
+++ b/psutil/arch/aix/net_kernel_structs.h
@@ -105,6 +105,6 @@ struct mbuf64
     struct m_hdr64 m_hdr;
 };

-#define m_len               m_hdr.mh_len
+#endif

-#endif
\ No newline at end of file
+#define m_len               m_hdr.mh_len
```

But, I assume this was inside the conditional for a reason, most likely that it will break on other AIX versions which I don't have access to test on.

I want to confirm that this worked for my AIX 7 for Python 2.7.15 installation - 

```
$ sudo python_64 setup.py sdist bdist_wheel
Password:
running sdist
running egg_info
writing requirements to psutil.egg-info/requires.txt
writing psutil.egg-info/PKG-INFO
writing top-level names to psutil.egg-info/top_level.txt
writing dependency_links to psutil.egg-info/dependency_links.txt
reading manifest file 'psutil.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'psutil.egg-info/SOURCES.txt'
running check
creating psutil-5.5.0
creating psutil-5.5.0/docs
creating psutil-5.5.0/docs/_static
creating psutil-5.5.0/docs/_static/css
creating psutil-5.5.0/psutil
creating psutil-5.5.0/psutil.egg-info
creating psutil-5.5.0/psutil/arch
creating psutil-5.5.0/psutil/arch/aix
creating psutil-5.5.0/psutil/arch/freebsd
creating psutil-5.5.0/psutil/arch/netbsd
creating psutil-5.5.0/psutil/arch/openbsd
creating psutil-5.5.0/psutil/arch/osx
creating psutil-5.5.0/psutil/arch/solaris
creating psutil-5.5.0/psutil/arch/solaris/v10
creating psutil-5.5.0/psutil/arch/windows
creating psutil-5.5.0/psutil/tests
creating psutil-5.5.0/scripts
creating psutil-5.5.0/scripts/internal
copying files to psutil-5.5.0...
copying .coveragerc -> psutil-5.5.0
copying .git-pre-commit -> psutil-5.5.0
copying .gitignore -> psutil-5.5.0
copying CREDITS -> psutil-5.5.0
copying DEVGUIDE.rst -> psutil-5.5.0
copying HISTORY.rst -> psutil-5.5.0
copying IDEAS -> psutil-5.5.0
copying INSTALL.rst -> psutil-5.5.0
copying LICENSE -> psutil-5.5.0
copying MANIFEST.in -> psutil-5.5.0
copying Makefile -> psutil-5.5.0
copying README.rst -> psutil-5.5.0
copying make.bat -> psutil-5.5.0
copying setup.cfg -> psutil-5.5.0
copying setup.py -> psutil-5.5.0
copying tox.ini -> psutil-5.5.0
copying docs/Makefile -> psutil-5.5.0/docs
copying docs/README -> psutil-5.5.0/docs
copying docs/conf.py -> psutil-5.5.0/docs
copying docs/index.rst -> psutil-5.5.0/docs
copying docs/make.bat -> psutil-5.5.0/docs
copying docs/_static/copybutton.js -> psutil-5.5.0/docs/_static
copying docs/_static/favicon.ico -> psutil-5.5.0/docs/_static
copying docs/_static/sidebar.js -> psutil-5.5.0/docs/_static
copying docs/_static/css/custom.css -> psutil-5.5.0/docs/_static/css
copying psutil/DEVNOTES -> psutil-5.5.0/psutil
copying psutil/__init__.py -> psutil-5.5.0/psutil
copying psutil/_common.py -> psutil-5.5.0/psutil
copying psutil/_compat.py -> psutil-5.5.0/psutil
copying psutil/_exceptions.py -> psutil-5.5.0/psutil
copying psutil/_psaix.py -> psutil-5.5.0/psutil
copying psutil/_psbsd.py -> psutil-5.5.0/psutil
copying psutil/_pslinux.py -> psutil-5.5.0/psutil
copying psutil/_psosx.py -> psutil-5.5.0/psutil
copying psutil/_psposix.py -> psutil-5.5.0/psutil
copying psutil/_pssunos.py -> psutil-5.5.0/psutil
copying psutil/_psutil_aix.c -> psutil-5.5.0/psutil
copying psutil/_psutil_bsd.c -> psutil-5.5.0/psutil
copying psutil/_psutil_common.c -> psutil-5.5.0/psutil
copying psutil/_psutil_common.h -> psutil-5.5.0/psutil
copying psutil/_psutil_linux.c -> psutil-5.5.0/psutil
copying psutil/_psutil_osx.c -> psutil-5.5.0/psutil
copying psutil/_psutil_posix.c -> psutil-5.5.0/psutil
copying psutil/_psutil_posix.h -> psutil-5.5.0/psutil
copying psutil/_psutil_sunos.c -> psutil-5.5.0/psutil
copying psutil/_psutil_windows.c -> psutil-5.5.0/psutil
copying psutil/_pswindows.py -> psutil-5.5.0/psutil
copying psutil.egg-info/PKG-INFO -> psutil-5.5.0/psutil.egg-info
copying psutil.egg-info/SOURCES.txt -> psutil-5.5.0/psutil.egg-info
copying psutil.egg-info/dependency_links.txt -> psutil-5.5.0/psutil.egg-info
copying psutil.egg-info/not-zip-safe -> psutil-5.5.0/psutil.egg-info
copying psutil.egg-info/requires.txt -> psutil-5.5.0/psutil.egg-info
copying psutil.egg-info/top_level.txt -> psutil-5.5.0/psutil.egg-info
copying psutil/arch/aix/common.c -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/common.h -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/ifaddrs.c -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/ifaddrs.h -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/net_connections.c -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/net_connections.h -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/aix/net_kernel_structs.h -> psutil-5.5.0/psutil/arch/aix
copying psutil/arch/freebsd/proc_socks.c -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/freebsd/proc_socks.h -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/freebsd/specific.c -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/freebsd/specific.h -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/freebsd/sys_socks.c -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/freebsd/sys_socks.h -> psutil-5.5.0/psutil/arch/freebsd
copying psutil/arch/netbsd/socks.c -> psutil-5.5.0/psutil/arch/netbsd
copying psutil/arch/netbsd/socks.h -> psutil-5.5.0/psutil/arch/netbsd
copying psutil/arch/netbsd/specific.c -> psutil-5.5.0/psutil/arch/netbsd
copying psutil/arch/netbsd/specific.h -> psutil-5.5.0/psutil/arch/netbsd
copying psutil/arch/openbsd/specific.c -> psutil-5.5.0/psutil/arch/openbsd
copying psutil/arch/openbsd/specific.h -> psutil-5.5.0/psutil/arch/openbsd
copying psutil/arch/osx/process_info.c -> psutil-5.5.0/psutil/arch/osx
copying psutil/arch/osx/process_info.h -> psutil-5.5.0/psutil/arch/osx
copying psutil/arch/solaris/environ.c -> psutil-5.5.0/psutil/arch/solaris
copying psutil/arch/solaris/environ.h -> psutil-5.5.0/psutil/arch/solaris
copying psutil/arch/solaris/v10/ifaddrs.c -> psutil-5.5.0/psutil/arch/solaris/v10
copying psutil/arch/solaris/v10/ifaddrs.h -> psutil-5.5.0/psutil/arch/solaris/v10
copying psutil/arch/windows/inet_ntop.c -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/inet_ntop.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/ntextapi.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/process_handles.c -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/process_handles.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/process_info.c -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/process_info.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/security.c -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/security.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/services.c -> psutil-5.5.0/psutil/arch/windows
copying psutil/arch/windows/services.h -> psutil-5.5.0/psutil/arch/windows
copying psutil/tests/README.rst -> psutil-5.5.0/psutil/tests
copying psutil/tests/__init__.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/__main__.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_aix.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_bsd.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_connections.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_contracts.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_linux.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_memory_leaks.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_misc.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_osx.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_posix.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_process.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_sunos.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_system.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_unicode.py -> psutil-5.5.0/psutil/tests
copying psutil/tests/test_windows.py -> psutil-5.5.0/psutil/tests
copying scripts/battery.py -> psutil-5.5.0/scripts
copying scripts/cpu_distribution.py -> psutil-5.5.0/scripts
copying scripts/disk_usage.py -> psutil-5.5.0/scripts
copying scripts/fans.py -> psutil-5.5.0/scripts
copying scripts/free.py -> psutil-5.5.0/scripts
copying scripts/ifconfig.py -> psutil-5.5.0/scripts
copying scripts/iotop.py -> psutil-5.5.0/scripts
copying scripts/killall.py -> psutil-5.5.0/scripts
copying scripts/meminfo.py -> psutil-5.5.0/scripts
copying scripts/netstat.py -> psutil-5.5.0/scripts
copying scripts/nettop.py -> psutil-5.5.0/scripts
copying scripts/pidof.py -> psutil-5.5.0/scripts
copying scripts/pmap.py -> psutil-5.5.0/scripts
copying scripts/procinfo.py -> psutil-5.5.0/scripts
copying scripts/procsmem.py -> psutil-5.5.0/scripts
copying scripts/ps.py -> psutil-5.5.0/scripts
copying scripts/pstree.py -> psutil-5.5.0/scripts
copying scripts/sensors.py -> psutil-5.5.0/scripts
copying scripts/temperatures.py -> psutil-5.5.0/scripts
copying scripts/top.py -> psutil-5.5.0/scripts
copying scripts/who.py -> psutil-5.5.0/scripts
copying scripts/winservices.py -> psutil-5.5.0/scripts
copying scripts/internal/README -> psutil-5.5.0/scripts/internal
copying scripts/internal/bench_oneshot.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/bench_oneshot_2.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/check_broken_links.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/download_exes.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/generate_manifest.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/print_announce.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/print_timeline.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/purge_installation.py -> psutil-5.5.0/scripts/internal
copying scripts/internal/winmake.py -> psutil-5.5.0/scripts/internal
Writing psutil-5.5.0/setup.cfg
Creating tar archive
removing 'psutil-5.5.0' (and everything under it)
running bdist_wheel
running build
running build_py
running build_ext
building 'psutil._psutil_aix' extension
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/_psutil_common.c -o build/temp.aix-7.1-2.7/psutil/_psutil_common.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/_psutil_posix.c -o build/temp.aix-7.1-2.7/psutil/_psutil_posix.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/_psutil_aix.c -o build/temp.aix-7.1-2.7/psutil/_psutil_aix.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/arch/aix/net_connections.c -o build/temp.aix-7.1-2.7/psutil/arch/aix/net_connections.o
psutil/arch/aix/net_connections.c: In function 'read_unp_addr':
psutil/arch/aix/net_connections.c:52:10: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
     uo = (int)(mb.m_hdr.mh_data - unp_addr);
          ^
psutil/arch/aix/net_connections.c: In function 'process_file':
psutil/arch/aix/net_connections.c:183:35: warning: passing argument 2 of 'read_unp_addr' makes integer from pointer without a cast [-Wint-conversion]
             if (read_unp_addr(Kd, unp.unp_addr, unix_laddr_str,
                                   ^~~
psutil/arch/aix/net_connections.c:37:1: note: expected 'KA_T {aka long long unsigned int}' but argument is of type 'struct mbuf *'
 read_unp_addr(
 ^~~~~~~~~~~~~
psutil/arch/aix/net_connections.c:194:35: warning: passing argument 2 of 'read_unp_addr' makes integer from pointer without a cast [-Wint-conversion]
             if (read_unp_addr(Kd, unp.unp_addr, unix_raddr_str,
                                   ^~~
psutil/arch/aix/net_connections.c:37:1: note: expected 'KA_T {aka long long unsigned int}' but argument is of type 'struct mbuf *'
 read_unp_addr(
 ^~~~~~~~~~~~~
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/arch/aix/common.c -o build/temp.aix-7.1-2.7/psutil/arch/aix/common.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/arch/aix/ifaddrs.c -o build/temp.aix-7.1-2.7/psutil/arch/aix/ifaddrs.o
/opt/freeware/lib64/python2.7/config/ld_so_aix /usr/bin/gcc -maix64 -pthread -bI:/opt/freeware/lib64/python2.7/config/python.exp -L. -L/usr/lib/threads -L/opt/freeware/lib64 -L/opt/freeware/lib -L/usr/lib/lib64 -L/usr/lib -Wl,-blibpath:/opt/freeware/lib64:/opt/freeware/lib:/usr/lib:/lib -Wl,-brtl build/temp.aix-7.1-2.7/psutil/_psutil_common.o build/temp.aix-7.1-2.7/psutil/_psutil_posix.o build/temp.aix-7.1-2.7/psutil/_psutil_aix.o build/temp.aix-7.1-2.7/psutil/arch/aix/net_connections.o build/temp.aix-7.1-2.7/psutil/arch/aix/common.o build/temp.aix-7.1-2.7/psutil/arch/aix/ifaddrs.o -L/opt/freeware/lib64 -lperfstat -o build/lib.aix-7.1-2.7/psutil/_psutil_aix.so
ld: 0711-224 WARNING: Duplicate symbol: global destructors keyed to 65535_0___dso_handle
ld: 0711-224 WARNING: Duplicate symbol: .global destructors keyed to 65535_0___dso_handle
ld: 0711-224 WARNING: Duplicate symbol: __dso_handle
ld: 0711-345 Use the -bloadmap or -bnoquiet option to obtain more information.
building 'psutil._psutil_posix' extension
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/_psutil_common.c -o build/temp.aix-7.1-2.7/psutil/_psutil_common.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/_psutil_posix.c -o build/temp.aix-7.1-2.7/psutil/_psutil_posix.o
/usr/bin/gcc -maix64 -pthread -fno-strict-aliasing -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -D_LINUX_SOURCE_COMPAT -I/usr/include -I/opt/freeware/include -I/opt/freeware/include/ncurses -DNDEBUG -D_GNU_SOURCE -fPIC -fno-strict-aliasing -fwrapv -DPSUTIL_POSIX=1 -DPSUTIL_VERSION=550 -DPSUTIL_AIX=1 -I/opt/freeware/include/python2.7 -c psutil/arch/aix/ifaddrs.c -o build/temp.aix-7.1-2.7/psutil/arch/aix/ifaddrs.o
/opt/freeware/lib64/python2.7/config/ld_so_aix /usr/bin/gcc -maix64 -pthread -bI:/opt/freeware/lib64/python2.7/config/python.exp -L. -L/usr/lib/threads -L/opt/freeware/lib64 -L/opt/freeware/lib -L/usr/lib/lib64 -L/usr/lib -Wl,-blibpath:/opt/freeware/lib64:/opt/freeware/lib:/usr/lib:/lib -Wl,-brtl build/temp.aix-7.1-2.7/psutil/_psutil_common.o build/temp.aix-7.1-2.7/psutil/_psutil_posix.o build/temp.aix-7.1-2.7/psutil/arch/aix/ifaddrs.o -L/opt/freeware/lib64 -o build/lib.aix-7.1-2.7/psutil/_psutil_posix.so
ld: 0711-224 WARNING: Duplicate symbol: global destructors keyed to 65535_0___dso_handle
ld: 0711-224 WARNING: Duplicate symbol: .global destructors keyed to 65535_0___dso_handle
ld: 0711-224 WARNING: Duplicate symbol: __dso_handle
ld: 0711-345 Use the -bloadmap or -bnoquiet option to obtain more information.
installing to build/bdist.aix-7.1/wheel
running install
running install_lib
creating build/bdist.aix-7.1
creating build/bdist.aix-7.1/wheel
creating build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/__init__.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_common.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_compat.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_exceptions.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psaix.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psbsd.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_pslinux.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psosx.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psposix.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_pssunos.py -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psutil_aix.so -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_psutil_posix.so -> build/bdist.aix-7.1/wheel/psutil
copying build/lib.aix-7.1-2.7/psutil/_pswindows.py -> build/bdist.aix-7.1/wheel/psutil
creating build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/__init__.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/__main__.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_aix.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_bsd.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_connections.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_contracts.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_linux.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_memory_leaks.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_misc.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_osx.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_posix.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_process.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_sunos.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_system.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_unicode.py -> build/bdist.aix-7.1/wheel/psutil/tests
copying build/lib.aix-7.1-2.7/psutil/tests/test_windows.py -> build/bdist.aix-7.1/wheel/psutil/tests
running install_egg_info
Copying psutil.egg-info to build/bdist.aix-7.1/wheel/psutil-5.5.0-py2.7.egg-info
running install_scripts
adding license file "LICENSE" (matched pattern "LICEN[CS]E*")
creating build/bdist.aix-7.1/wheel/psutil-5.5.0.dist-info/WHEEL
creating 'dist/psutil-5.5.0-cp27-cp27m-aix_7_1.whl' and adding 'build/bdist.aix-7.1/wheel' to it
adding 'psutil/__init__.py'
adding 'psutil/_common.py'
adding 'psutil/_compat.py'
adding 'psutil/_exceptions.py'
adding 'psutil/_psaix.py'
adding 'psutil/_psbsd.py'
adding 'psutil/_pslinux.py'
adding 'psutil/_psosx.py'
adding 'psutil/_psposix.py'
adding 'psutil/_pssunos.py'
adding 'psutil/_psutil_aix.so'
adding 'psutil/_psutil_posix.so'
adding 'psutil/_pswindows.py'
adding 'psutil/tests/__init__.py'
adding 'psutil/tests/__main__.py'
adding 'psutil/tests/test_aix.py'
adding 'psutil/tests/test_bsd.py'
adding 'psutil/tests/test_connections.py'
adding 'psutil/tests/test_contracts.py'
adding 'psutil/tests/test_linux.py'
adding 'psutil/tests/test_memory_leaks.py'
adding 'psutil/tests/test_misc.py'
adding 'psutil/tests/test_osx.py'
adding 'psutil/tests/test_posix.py'
adding 'psutil/tests/test_process.py'
adding 'psutil/tests/test_sunos.py'
adding 'psutil/tests/test_system.py'
adding 'psutil/tests/test_unicode.py'
adding 'psutil/tests/test_windows.py'
adding 'psutil-5.5.0.dist-info/LICENSE'
adding 'psutil-5.5.0.dist-info/METADATA'
adding 'psutil-5.5.0.dist-info/WHEEL'
adding 'psutil-5.5.0.dist-info/top_level.txt'
adding 'psutil-5.5.0.dist-info/RECORD'
removing build/bdist.aix-7.1/wheel
```

Providing details for your own use.

--- a/psutil/arch/aix/net_kernel_structs.h
+++ b/psutil/arch/aix/net_kernel_structs.h
@@ -18,6 +18,7 @@
 #include <sys/protosw.h>
 #include <sys/unpcb.h>
 #include <sys/mbuf_base.h>
+#include <sys/mbuf_macro.h>
 #include <netinet/ip_var.h>
 #include <netinet/tcp.h>
 #include <netinet/tcpip.h>
@@ -30,7 +31,7 @@
 #define tcpcb64 tcpcb
 #define unpcb64 unpcb
 #define mbuf64 mbuf
-#else
+#else       /* __64BIT__ */
   struct file64 {
     int f_flag;
     int f_count;
@@ -107,4 +108,4 @@ struct mbuf64
 
 #define m_len               m_hdr.mh_len
 
-#endif
+#endif      /* __64BIT__ */

